handler_helo_test.go

```diff
@@ -19,7 +19,7 @@ func TestNewHandlerHelo(t *testing.T) {
 }
 
 func TestHandlerHeloRun(t *testing.T) {
-	t.Run("when successful HELO request", func(t *testing.T) {
+	t.Run("when successful HELO domain request", func(t *testing.T) {
```

Comment:
Honestly it's a wrong place for testing this cases. It's places for general cases of handler `helo` runner. You should cover all possible cases in `TestHandlerHeloIsInvalidCmdArg`, in line 143. Please cover cases for valid and invalid ip addresses. And seems like I have missed out case for `localhost` 🐒, so this case too please 😃

Also we can double check it here `handler_helo_test.go:236`:
```go
heloDomains := []string{"example.com", "localhost", "1.2.3.4", "[1.2.3.4]"}
```

Reply:
I agree, I should have looked lower in the file.
I will move them to the correct place
Thank you